src/pages/ProductDetail/ProductDetail.tsx

```diff
@@ -1,25 +1,38 @@
-import React, { useEffect, useCallback, useState } from 'react'
+import React, { useEffect, useCallback, useState } from 'react';
 import { useParams, useNavigate } from 'react-router-dom';
 import Header from '../../components/Header/Header';
-import styles from'./ProductDetail.module.css';
+import styles from './ProductDetail.module.css';
 import ProductInfo from './components/ProductInfo/ProductInfo';
 import { getProductDetail } from '../../api/itemApi';
 import Comment from './components/Comment/Comment';
 import backicon from '../../assets/images/backicon.png';
 
+export interface Item {
+  id: number;
+  name: string;
+  description: string;
+  price: number;
+  tags: string[];
+  images: string[];
+  ownerId: number;
+  ownerNickname: string;
+  favoriteCount: number;
+  createdAt: string;
+}
+
 function ProductDetail() {
-  const { productId } = useParams();
-  const [item, setItem] = useState(null);
+  const { productId } = useParams<{ productId :string}>();
+  const [item, setItem] = useState<Item | undefined>(undefined);
```

Comment:
P3:
아래 코드도 동일동작합니다~
```suggestion
  const [item, setItem] = useState<Item>();
```